_includes/fe-plugin-intro.md

```diff
@@ -0,0 +1,17 @@
+Frontend plugins provide a way to change the behavior of Deck, Spinnaker's UI
+service. You can add configuration and validation for new stages provided by
+Orca plugins, override existing components with your own implementation, or add
+new Kind definitions for custom resources in your environment.
+
+Plugins are written in any Javascript compatible language (Typescript included),
```

Comment:
s/Javascript/JavaScript and s/Typescript/TypeScript

Reply:
This is true but it might be worth mentioning that the pluginsdk's build tools are very JS/TS centric and you will be going off the deep end with coffeescript or kotlinjs.

Reply:
- [ ] casing on JS/TS
- [ ] tooling caveat